castle.js

```diff
@@ -0,0 +1,32 @@
+import {SPECS} from 'battlecode';
```

Comment:
Suggestion: We can import SPECS from bc19 instead of battlecode. 
As battlecode is a dependecy on bc19.